src/Symfony/Component/Form/Tests/Extension/Core/Type/ChoiceTypeTest.php

```diff
@@ -1194,7 +1226,7 @@ public function testSubmitSingleExpandedNonRequiredFalse()
         $form->submit(false);
 
         $this->assertNull($form->getData());
-        $this->assertNull($form->getViewData());
+        $this->assertEmpty($form->getViewData());
```

Comment:
I see. You should rather use `assertSame('', ...)` though if you want to ensure this is an empty string. `assertEmpty()` accepts a lot of values (`null`, `false`, ...).